config/scout.php

```diff
@@ -41,7 +41,7 @@
     |
     */
 
-    'queue' => env('SCOUT_QUEUE', false),
+    'queue' => env('SCOUT_QUEUE', true),
```

Comment:
Since Rogue already has queue's set up, I didn't need to set them up and could just turn this on, which is recommended!